tests/integration/components/dashboard-agenda-test.js

```diff
@@ -47,6 +53,48 @@ module('Integration | Component | dashboard agenda', function (hooks) {
         slug: 'whatever',
         postrequisiteSlug: 'something',
         postrequisiteName: 'first',
+        isPublished: true,
+        isScheduled: false,
+        isBlanked: false,
+      },
+      {
+        name: 'blanked prework',
+        startDate: today.format(),
+        endDate: tomorrow.format(),
+        location: "room 111",
+        ilmSession: true,
+        slug: 'whatever',
+        postrequisiteSlug: 'something',
+        postrequisiteName: 'first',
+        isPublished: true,
+        isScheduled: false,
+        isBlanked: true,
+      },
+      {
+        name: 'scheduled prework',
+        startDate: today.format(),
+        endDate: tomorrow.format(),
+        location: "room 111",
+        ilmSession: true,
+        slug: 'whatever',
+        postrequisiteSlug: 'something',
+        postrequisiteName: 'first',
+        isPublished: true,
+        isScheduled: true,
+        isBlanked: false,
+      },
+      {
+        name: 'unpublished prework',
+        startDate: today.format(),
+        endDate: tomorrow.format(),
+        location: "room 111",
+        ilmSession: true,
+        slug: 'whatever',
+        postrequisiteSlug: 'something',
+        postrequisiteName: 'first',
+        isPublished: true,
+        isScheduled: true,
+        isBlanked: false,
```

Comment:
I like it when adding tests just means adding fixtures, the tests are already working!